entity-framework/core/providers/cosmos/unstructured-data.md

```diff
@@ -44,11 +43,11 @@ It is possible to access the properties that are not tracked by EF Core through
 
 ## Using CosmosClient
 
-To decouple completely from EF Core get the `CosmosClient` object that is [part of the Azure Cosmos DB SDK](https://docs.microsoft.com/en-us/azure/cosmos-db/sql-api-get-started) from `DbContext`:
+To decouple completely from EF Core get the `CosmosClient` object that is [part of the Azure Cosmos DB SDK](https://docs.microsoft.com/azure/cosmos-db/sql-api-get-started) from `DbContext`:
 
 [!code-csharp[CosmosClient](../../../../samples/core/Cosmos/UnstructuredData/Sample.cs?highlight=3&name=CosmosClient)]
 
-## Missing property values
+## Missing Property Values
```

Comment:
I remember one of my Neuroscience processors telling me not to case sub-titles like this. Maybe he was wrong? Maybe thinks have changed? @bricelam? 🚎

Reply:
I'll revert.